src/main/java/org/codeforamerica/shiba/output/caf/NonHourlyJobIncomeInformation.java

```diff
@@ -1,18 +1,19 @@
 package org.codeforamerica.shiba.output.caf;
 
 import lombok.Value;
+import org.codeforamerica.shiba.Money;
 import org.codeforamerica.shiba.pages.data.Iteration;
 
 @Value
 public class NonHourlyJobIncomeInformation implements JobIncomeInformation {
     PayPeriod payPeriod;
-    Double incomePerPayPeriod;
+    Money incomePerPayPeriod;
     int indexInJobsSubworkflow;
     Iteration iteration;
 
     public NonHourlyJobIncomeInformation(String payPeriod, String incomePerPayPeriod, int indexInJobsSubworkflow, Iteration iteration) {
         this.payPeriod = payPeriod.isEmpty() ? null : PayPeriod.valueOf(payPeriod);
-        this.incomePerPayPeriod = incomePerPayPeriod.isEmpty() ? null : Double.valueOf(incomePerPayPeriod.replace(",",""));
+        this.incomePerPayPeriod = incomePerPayPeriod.isEmpty() ? null : new Money(parseWithCommasRemoved(incomePerPayPeriod));
```

Comment:
I know it's not your fault we used null here but I'm wondering if we can use $0 instead of null